iothub/device/src/DeviceClient.ConventionBasedOperations.cs

```diff
@@ -39,28 +39,55 @@ public Task SendTelemetryAsync(TelemetryMessage telemetryMessage, CancellationTo
             => InternalClient.SendTelemetryAsync(telemetryMessage, cancellationToken);
 
         /// <summary>
-        /// Sets the listener for command invocation requests.
+        /// Sets the listener for command calls from the service.
         /// </summary>
         /// <param name="callback">The callback to handle all incoming commands for the client.</param>
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
+        /// <example>
+        /// Inline:
+        /// <code language="csharp">
+        /// await client.SubscribeToCommandsAsync(
+        ///     (commandRequest) =>
+        ///     {
+        ///         switch (commandRequest.CommandName)
```

Comment:
Maybe have this be a check for a specific command name so that you can demonstrate returning 200 within this block and 400 outside of it